packages/chrome/styleguide.config.js

```diff
@@ -5,42 +5,72 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+const path = require('path');
+const reactDocgenTypescript = require('react-docgen-typescript');
+const reactDocgen = require('react-docgen');
+
 /**
  * Package specific styleguide configuration
  * https://github.com/styleguidist/react-styleguidist/blob/master/docs/Configuration.md
  */
 module.exports = {
+  propsParser: reactDocgenTypescript.withCustomConfig(
+    path.resolve(__dirname, '../../tsconfig.json'),
+    {
+      propFilter: props => {
+        return props.parent.fileName.indexOf('node_modules') === -1;
+      }
+    }
+  ).parse,
+  resolver: reactDocgen.resolver.findAllComponentDefinitions,
   sections: [
     {
       name: '',
       content: '../../packages/chrome/README.md'
     },
     {
-      name: 'Views',
+      name: 'Examples',
+      sections: [
+        {
+          name: 'Basic',
+          content: '../../packages/chrome/examples/basic.md'
+        },
+        {
+          name: 'Header',
+          content: '../../packages/chrome/examples/header.md'
+        },
+        {
+          name: 'Sidebar',
+          content: '../../packages/chrome/examples/sidebar.md'
+        }
+      ]
+    },
+    {
+      name: 'Elements',
       sections: [
         {
           name: 'Chrome',
-          components: '../../packages/chrome/src/views/[A-Z]*.js'
+          components: '../../packages/chrome/src/elements/[A-Z]*.{ts,tsx}'
         },
         {
           name: 'Nav',
-          components: '../../packages/chrome/src/views/nav/[A-Z]*.js'
+          components: '../../packages/chrome/src/elements/nav/[A-Z]*.{ts,tsx}'
         },
         {
           name: 'SubNav',
-          components: '../../packages/chrome/src/views/subnav/[A-Z]*.js'
+          components: '../../packages/chrome/src/elements/subnav/[A-Z]*.{ts,tsx}'
         },
         {
           name: 'Body',
-          components: '../../packages/chrome/src/views/body/[A-Z]*.js',
+          components: '../../packages/chrome/src/elements/body/[A-Z]*.{ts,tsx}',
           sections: [
             {
               name: 'Header',
-              components: '../../packages/chrome/src/views/header/[A-Z]*.js'
+              components: '../../packages/chrome/src/elements/header/[A-Z]*.{ts,tsx}'
             },
             {
               name: 'Footer',
-              components: '../../packages/chrome/src/views/footer/[A-Z]*.js'
+              components: '../../packages/chrome/src/elements/footer/[A-Z]*.{ts,tsx}'
```

Comment:
Specify body sections so that they read in alpha order: Body, Content, Footer, Header, Main, Sidebar.

Reply:
From our offline conversation, I've updated this to be alphabetical through the folders and individual components.